src/app/forecasting/page.tsx

```
@@ -0,0 +1,146 @@
+"use client";
+
+import React, { useState, useEffect } from 'react';
+import RevenueForecast from '../../components/forecasting/RevenueForecast';
+
+interface RevenueForecast {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'RevenueForecast'. Consider to delete it or rename it.

'RevenueForecast' is defined here:


(lint/suspicious/noRedeclare)